apis/r/tests/testthat/test-shape.R

```diff
@@ -4,17 +4,20 @@ test_that("SOMADataFrame shape", {
   asch <- create_arrow_schema()
 
   index_column_name_choices = list(
-    "soma_joinid",
-    c("soma_joinid", "foo"),
-    c("soma_joinid", "baz"),
-    c("baz", "foo")
+    one = "soma_joinid",
+    two = c("soma_joinid", "foo"),
+    three = c("soma_joinid", "baz"),
+    four = c("baz", "foo")
   )
 
-  for (index_column_names in index_column_name_choices) {
+  for (case_name in names(index_column_name_choices)) {
+    index_column_names = index_column_name_choices[[case_name]]
```

Comment:
These test modifications can be simplified:
 - we're iterating through a list and performing the same sets of checks, we can use `seq_along()` to iterate along the list for us
```suggestion
  for (i in seq_along(index_column_name_choices)) {
    index_column_names <- index_column_name_choices[[i]]
```